Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -594,18 +689,17 @@ def xdr_ioc_to_demisto(ioc: dict) -> dict:
 
 def get_changes(client: Client):
     demisto.debug("pull XDR changes: starting")
-    from_time: dict = get_integration_context()
-    if not from_time:
+    integration_context: dict = get_integration_context()
+    if not integration_context:
         raise DemistoException('XDR is not synced.')
-    path, requests_kwargs = prepare_get_changes(from_time['ts'])
+    path, requests_kwargs = prepare_get_changes(integration_context['ts'])
     requests_kwargs: dict = get_requests_kwargs(_json=requests_kwargs)
     demisto.debug(f'pull XDR changes: calling endpoint {path}, {requests_kwargs=}')
     if iocs := client.http_request(url_suffix=path, requests_kwargs=requests_kwargs).get('reply', []):
-        from_time['ts'] = iocs[-1].get('RULE_MODIFY_TIME', from_time) + 1
-        demisto.info(f'pull XDR changes: got {len(iocs)} IOCs from XDR')
-        demisto.info(f'pull XDR changes: setting {from_time} to integration context ')
-        set_integration_context(from_time)
-        demisto.debug(f"pull XDR changes: converting {len(iocs)} XDR IOCs to demisto format, then creating indicators")
+        integration_context['ts'] = iocs[-1].get('RULE_MODIFY_TIME', integration_context) + 1
```

Comment:
not sure Im understand. why the default is integration_context?

Reply:
changed to
last_run['ts'] = iocs[-1].get('RULE_MODIFY_TIME', last_run['ts']) + 1